pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "pipeline-ai"
-version = "0.3.11"
+version = "0.3.12"
```

Comment:
As this feature is included under the v0.4.0 spec should this be reflecting that?

Reply:
Perhaps yes, although to me the real feature is _creating_ environments, which this PR does not expose.

As this PR includes a schema change, if we want for the 0.4.0 release we won't be able to test the 'read' aspect of the API (i.e. this default environment behaviour) until everything is merged.